docs/reference/taxii-feeds.md

```diff
@@ -1,7 +1,32 @@
 # Taxii feeds
+In OpenCTI, you can import TAXII collections from the Data tab, then Ingestion under the TAXII Feeds tab.
 
-!!! tip "Under construction"
+![TAXII Feeds panel.png](assets/TAXII_Feeds_panel.png)
 
-    We are doing our best to complete this page. 
-    If you want to participate, don't hesitate to join the [Filigran Community on Slack](https://community.filigran.io) 
-    or submit your pull request on the [Github doc repository](https://github.com/OpenCTI-Platform/docs).
+## TAXII Feeds management interface
```

Comment:
feeds in lower case

Reply:
same in the the first line of the following text

Reply:
Sure? Cause within the platform the 'F' is in uppercase: 'TAXII Feeds' so i wasn't sure